pkg/apis/sources/v1alpha1/cron_job_types.go

```diff
@@ -55,13 +56,15 @@ var (
 const (
 	// CronJobEventType is the CronJob CloudEvent type.
 	CronJobEventType = "dev.knative.cronjob.event"
-	// CronJobEventSource is the CronJob CloudEvent source.
-	CronJobEventSource = "/CronJob"
 )
 
+// CronJobEventSource returns the CronJob CloudEvent source.
+func CronJobEventSource(cronJobName string) string {
+	return fmt.Sprintf("urn:cronjob:%s", cronJobName)
```

Comment:
I think we should use the cluster local url ref for this because we have no other usage of urn, and we do not know the namespace from this link.

Reply:
Does source have to be a URL? Isn't it that it has to be a URI Ref, and that includes urns? And we as producers are defining that urn:cronjob is the namespace for cron jobs?

Reply:
it has to be a url ref is the requirement. I am saying we do not use urn anywhere else yet and you can't use it to find the pod that is making the event (unless there is a feature I do not know about.) I would expect `source: /apis/v1/namespaces/default/pods/cronjon-rhx9r-7957589cbc-cgq9t`

Reply:
or even `source: /apis/v1/namespaces/default/cronjobsources/cronjon-rhx9r`

Reply:
> url ref

I thought it was a URI ref, but might be wrong...

> /apis/v1/namespaces/default/cronjobsources/cronjon-rhx9r

OK, will go with this one... Thanks!

Reply:
Done